casbin/model/policy.py

```diff
@@ -85,10 +85,30 @@ def add_policies(self,sec,ptype,rules):
     def update_policy(self, sec, ptype, old_rule, new_rule):
         """update a policy rule from the model."""
 
-        if not self.has_policy(sec, ptype, old_rule):
+        if sec not in self.model.keys():
+            return False
+        if ptype not in self.model[sec]:
+            return False
+
+        ast = self.model[sec][ptype]
+
+        # find rule index, if fail return false
+        try:
+            rule_index = ast.policy.index(old_rule)
+        except:
             return False
+        # find rule priority index, if fail, add directly.
+        try:
+            priority_index = ast.tokens.index(old_rule)
```

Comment:
it should be `priority_index = ast.tokens.index(old_rule)`

Reply:
And if possible, plz add unit tests for `update_policy` with priority policies.

Reply:
And it will be more elegant with `in` operater. like `if "p_priority" in ast.tokens`

Reply:
Thanks for your advices. I am very glad to add unit tests for update_policy with priority policies, but I don't know which model uses this property after my hours of search.